backend/src/tasks/hibp.ts

```diff
@@ -40,30 +63,38 @@ async function lookupEmails(breachesDict: any, domain: Domain) {
       }
     ).json();
 
-    const AddressResults = {};
-    const BreachResults = {};
+    const addressResults = {};
+    const breachResults = {};
     const finalResults = {};
 
     const shouldCountBreach = (breach) =>
-      breach.DataClasses.indexOf('Passwords') > -1 &&
-      breach.IsVerified === true &&
-      breach.BreachDate > '2016-01-01';
+      breach.IsVerified === true && breach.BreachDate > '2016-01-01';
 
     for (const email in results) {
       const filtered = (results[email] || []).filter((e) =>
         shouldCountBreach(breachesDict[e])
       );
       if (filtered.length > 0) {
-        AddressResults[email + '@' + domain.name] = filtered;
+        addressResults[email + '@' + domain.name] = filtered;
         for (const breach of filtered) {
-          if (!(breach in BreachResults)) {
-            BreachResults[breach] = breachesDict[breach];
+          if (!(breach in breachResults)) {
+            breachResults[breach] = breachesDict[breach];
+            breachResults[breach].passwordIncluded =
+              breachResults[breach].DataClasses.indexOf('Passwords') > -1;
           }
         }
       }
     }
-    finalResults['Emails'] = AddressResults;
-    finalResults['Breaches'] = BreachResults;
+
+    // for (const breach in breachResults) {
```

Comment:
Did you mean to comment this out?

Reply:
Sorry, I commented out the part that you said to remove, to test to make sure it would work before I deleted it and forgot, I just wanted to verify that this was the part you were saying that I should remove

Reply:
let's remove it